src/main/java/entity/Payment.java

```diff
@@ -0,0 +1,80 @@
+package entity;
+
+import javax.persistence.*;
+
+@Entity
+public class Payment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private String payer;
+
```

Comment:
Because, as I thing, String type, makes more sense, if not only registered users can donate,...